modules/kafka/src/test/java/com/devonfw/module/consumer/KafkaMessageReceiverTest.java

```diff
@@ -0,0 +1,78 @@
+package com.devonfw.module.consumer;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.test.EmbeddedKafkaBroker;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+import org.springframework.kafka.test.utils.KafkaTestUtils;
+
+import com.devonfw.module.common.config.KafkaConsumerProperties;
+import com.devonfw.module.test.common.base.ModuleTest;
+
+/**
+ * @author Ravichandran
+ *
+ */
+
+@EmbeddedKafka
+public class KafkaMessageReceiverTest extends ModuleTest {
+
+  private static final String TOPIC = "test_topic";
+
+  private EmbeddedKafkaBroker embeddedKafkaBroker;
+
+  private KafkaMessageReceiver KafkaMessageReceiver;
+
+  /**
+   *
+   */
+  @BeforeEach
+  public void setup() {
+
+    this.KafkaMessageReceiver = new KafkaMessageReceiver();
+
+    this.embeddedKafkaBroker = new EmbeddedKafkaBroker(1, true, TOPIC);
+  }
+
+  /**
+   *
+   */
```

Comment:
You do not need to add JavaDoc here. But if you do then also fill it. Otherwise this is just a waste of lines of code.